plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1090,7 +1108,14 @@ func (c *PodAffinityChecker) satisfiesPodsAffinityAntiAffinity(pod *v1.Pod, node
 			// If the requirement matches a pod's own labels are namespace, and there are
 			// no other such pods, then disregard the requirement. This is necessary to
 			// not block forever because the first pod of the collection can't be scheduled.
-			match, err := priorityutil.PodMatchesTermsNamespaceAndSelector(pod, pod, &term)
+			selector, err := metav1.LabelSelectorAsSelector(term.LabelSelector)
+			if err != nil {
+				glog.V(10).Infof("Cannot schedule pod  %+v onto node %v, because of error creating selector for term %v, err: %v",
```

Comment:
This error should be about not able to parse selector, not about not being able to schedule.